Update agama flavors in staging test

lemon-suse opened the pull request:
Related ticket: https://progress.opensuse.org/issues/133955

sofiasyria commented:
Shouldn't we have 4 images for 3 archs each? 
https://build.opensuse.org/project/monitor/systemsmanagement:Agama:Staging?arch_x86_64=1&blocked=1&broken=1&building=1&defaults=0&deleting=1&dispatching=1&failed=1&finished=1&locked=1&repo_images=1&scheduled=1&signing=1&succeeded=1&unresolvable=1

andrii-suse commented:
> Yes, we should have, but it seems some build issue there.

Nope, you must manually modify t/obs/systemsmanagement:Agama:Staging/files_iso.lst and remove old items and add expected new files to see corresponding output from the tests.

andrii-suse commented:
This is current output, you can put it into files_iso.lst and commit all changes after `make test_update_before_files` : 
```
> rsync obspublish-other::openqa/systemsmanagement:Agama:Staging//images/*/*/*.iso | awk '{ $1=$2=$3=$4=""; print substr($0,5); }'
agama-live.aarch64-3.0.0-ALP-Build2.26.iso
agama-live.aarch64-3.0.0-ALP-Playwright-Build2.27.iso
agama-live.aarch64-3.0.0-openSUSE-Build2.31.iso
agama-live.aarch64-3.0.0-openSUSE-Playwright-Build2.27.iso
agama-live.ppc64le-3.0.0-openSUSE-Build2.31.iso
agama-live.s390x-3.0.0-ALP-Build2.24.iso
agama-live.s390x-3.0.0-openSUSE-Build2.26.iso
agama-live.x86_64-3.0.0-ALP-Build2.26.iso
agama-live.x86_64-3.0.0-ALP-Playwright-Build2.27.iso
agama-live.x86_64-3.0.0-openSUSE-Build2.31.iso
agama-live.x86_64-3.0.0-openSUSE-Playwright-Build2.27.iso
```

lemon-suse commented:
> This is current output, you can put it into files_iso.lst and commit all changes after `make test_update_before_files` :
> 
> ```
> > rsync obspublish-other::openqa/systemsmanagement:Agama:Staging//images/*/*/*.iso | awk '{ $1=$2=$3=$4=""; print substr($0,5); }'
> agama-live.aarch64-3.0.0-ALP-Build2.26.iso
> agama-live.aarch64-3.0.0-ALP-Playwright-Build2.27.iso
> agama-live.aarch64-3.0.0-openSUSE-Build2.31.iso
> agama-live.aarch64-3.0.0-openSUSE-Playwright-Build2.27.iso
> agama-live.ppc64le-3.0.0-openSUSE-Build2.31.iso
> agama-live.s390x-3.0.0-ALP-Build2.24.iso
> agama-live.s390x-3.0.0-openSUSE-Build2.26.iso
> agama-live.x86_64-3.0.0-ALP-Build2.26.iso
> agama-live.x86_64-3.0.0-ALP-Playwright-Build2.27.iso
> agama-live.x86_64-3.0.0-openSUSE-Build2.31.iso
> agama-live.x86_64-3.0.0-openSUSE-Playwright-Build2.27.iso
> ```

Thanks, changed and updated PR.

sofiasyria commented on xml/obs/systemsmanagement:Agama:Staging.xml:
```diff
@@ -7,10 +7,13 @@
+    <flavor name="agama-live-openSUSE" folder="images/*/agama-live:openSUSE">
+        <hdd filemask="agama-live.*-openSUSE-Build.*iso$"/>
+    </flavor>
+    <flavor name="agama-live-openSUSE-Playwright" folder="images/*/agama-live:openSUSEi-Playwright">
```
There is a typo here: s/openSUSEi/openSUSE/g , let's do this by using the scripts, because manually we don't know what mistake we might do.

lemon-suse replied:
Thanks, updated.

andrii-suse reviewed (commented):
Should we also change version= in the XML file? Because current binaries have 3.0, not 2.1

andrii-suse commented:
> Should we also change version= in the XML file? Because current binaries have 3.0, not 2.1

Ah it was changed in #222 , so just rebase to current master.

lemon-suse commented:
> > Should we also change version= in the XML file? Because current binaries have 3.0, not 2.1
> 
> Ah it was changed in #222 , so just rebase to current master.

Thanks, we can close this MR then.

lemon-suse commented:
Already merged for https://github.com/os-autoinst/openqa-trigger-from-obs/pull/222, so we can close this PR.
